app/views/SelectServerView.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+	FlatList, View, StyleSheet
+} from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { SafeAreaView } from 'react-navigation';
+
+import I18n from '../i18n';
+import database from '../lib/realm';
+import StatusBar from '../containers/StatusBar';
+import { selectServerRequest as selectServerRequestAction } from '../actions/server';
+
+import {
+	COLOR_BACKGROUND_CONTAINER, COLOR_WHITE
+} from '../constants/colors';
+import Navigation from '../lib/Navigation';
+import ServerItem from '../presentation/ServerItem';
+
+const getItemLayout = (data, index) => ({ length: 68, offset: 68 * index, index });
```

Comment:
I think we should export ServerItem height as const and import here.
RoomItem does the same.
We declare in RoomItem's styles and export it again on RoomItem's index.
It makes possible to do this later on RoomsListView:
```
import RoomItem, { ROW_HEIGHT } from '../../presentation/RoomItem';
```
This way we can make sure `getItemLayout` will work 1 year from now.